test-jest/cartesian/CartesianAxis.spec.tsx

```diff
@@ -66,29 +76,36 @@ describe('<CartesianAxis />', () => {
       </Surface>,
     );
 
-    expect(wrapper.find('.recharts-cartesian-axis-tick').length).to.equal(5);
+    expect(container.querySelectorAll('.recharts-cartesian-axis-tick')).toHaveLength(5);
   });
 
   it('gets font states from its ComputedStyle', () => {
-    const stub = sinon.stub(window, 'getComputedStyle').returns({ fontSize: '14px', letterSpacing: '0.5em' });
-    const wrapper = mount(
-      <CartesianAxis
-        orientation="bottom"
-        width={400}
-        height={50}
-        viewBox={{ x: 0, y: 0, width: 500, height: 500 }}
-        ticks={ticks}
-      />,
-    );
+    const myStyle = { fontSize: '14px', letterSpacing: '0.5em' } as CSSStyleDeclaration;
```

Comment:
nit: could this be `const myStyle: CSSStyleDeclaration` instead of a cast?

Reply:
It is simply not possible. If I want to make it possible, I must declare a massive object that contains every property from CSSStyleDeclaration.

I  am new to TypeScript, Is there anything wrong with casting?

Reply:
You can do 
```typescript
const myStyle: Partial<CSSStyleDeclaration
```

This way you don't have to cast and not all the attributes are required. Casting is unsafe as you could have an incorrect attribute in the object and you are basically saying "I am right" with the cast that you make. I'll merge this though, its fine in a test